templates/tfengine/recipes/devops.hcl

```diff
@@ -52,7 +52,7 @@ schema = {
       description = "Location of state bucket."
       type        = "string"
     }
-    org_admin = {
+    admin = {
```

Comment:
If this is always group, I think it would be clearer to rename to `admin_group`.

Reply:
Good idea, done